app/models/lockstep/user.rb

```diff
@@ -7,4 +7,27 @@ class Lockstep::User < Lockstep::ApiRecord
   belongs_to :account, class_name: "Lockstep::Account", loader: ->(user) { Lockstep::Account.first }
 
   alias_attribute :last_logged_in, :b2_c_last_logged_in
+
+  def self.invite(email_addresses)
+    emails = email_addresses.map { |email| { email: email } }
+    resp = resource.post('/invite', body: emails, params: {})
+    parsed_response = JSON.parse(resp.body)
+    if resp.code != '200'
+      if resp.code == '401'
+        raise Lockstep::Exceptions::UnauthorizedError, parsed_response['title']
+      elsif resp.code == '400'
+        raise Lockstep::Exceptions::BadRequestError, parsed_response['title']
+      elsif resp.code == '404'
+        raise Lockstep::Exceptions::RecordNotFound, parsed_response['title']
+      end
+    end
+    result = parsed_response&.map do |user|
+      if user['success']
+        user['invitedUser'] = Lockstep::User.find_by(user_id: user['invitedUser']['userId'])
```

Comment:
If we invite 20 users, this will call the API 20 times. Very inefficient.

Do we need to make an API call to fetch the user details? Since we have the entire object in `user`, can't we just initialise a new `Lockstep::User` with the attributes without making an API call?

Reply:
Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/b9f58081b23d7de7ba154373d4b87db2165b0c70)